src/Protocol/MojangAPI.cpp

```diff
@@ -440,13 +440,13 @@ bool cMojangAPI::SecureRequest(const AString & a_ServerName, const AString & a_R
 	Socket.SetExpectedPeerName(a_ServerName);
 	if (!Socket.Connect(a_ServerName, 443))
 	{
-		LOGWARNING("%s: Can't connect to %s: %s", __FUNCTION__, a_ServerName.c_str(), Socket.GetLastErrorText().c_str());
+		LOGWARNING("%s: Can't connect to %s: %s", __METHOD_NAME__, a_ServerName.c_str(), Socket.GetLastErrorText().c_str());
```

Comment:
I think this will not work properly, `LOGWARNING` expects a `char *`, but `__METHOD_NAME__` provides a `std::string_view`

Reply:
With gcc this works: 

`Log: [1f9f141165174f7c|22:16:56] cRoot::Run, Startup complete, took 3036ms!
`

check at a place which is defenetly run

Reply:
Works, but is undefined behavior, so...

Reply:
I'll check with MSVC, just for the fun.

Reply:
Oh, `LOGWARNING` et al. have been converted to `fmt` already, so despite accepting printf-like format strings, they work with `std::string` etc.
Right, my bad. Sorry.